GOVERNANCE.md

```diff
@@ -42,6 +45,33 @@ Maintainers have right to nominate contributors to become maintainers and vote o
 
 Individuals can be maintainers to multiple Eiffel Community Open Source Projects.
 
+
+## Community Roles
+
+Eiffel Community currently has two roles relevant to community governance and support.
+
+### Election Officers
+
+Eiffel Community Election Officers are the community members who are appointed by Eiffel Technical Committee to conduct community elections.
+
+The term for Election Officers is **one year**.
+
+Responsibilities of Election Officers are
+
+* Determine the list of community members who are eligible to vote and run in community elections.
+* Conduct the elections based on the process documented in [Election Process](#Election-Process).
+* Ensure the elections are done in a transparent way.
+* Guide the Technical Committee and Eiffel Community at large in election matters.
+
+Eiffel Community Election Officers are the individuals listed below:
+
+* TBD
+* TBD
+
+### Security Officers
+
+TBD
+
 ## Technical Committee
 
 The Technical Committee is responsible for the oversight of the Eiffel Community Open Source Projects. Technical Committee is comprised of **X** voting members.
```

Comment:
We need to set this number. I think 7 was brought up during the 11th of March BCT meeting.

Reply:
Thanks for putting this here. I've forgotten about the date where we decided this.